build/packer/pmm2.json

```diff
@@ -136,7 +136,7 @@
         {
             "type": "ansible-local",
             "playbook_file": "ansible/pmm2/post-build-actions.yml",
-            "extra_arguments": ["-v"]
+            "extra_arguments": ["-vvv"]
```

Comment:
That type of verbosity is almost impossible to read. Is it a leftover?

Reply:
Since 'update.yml' already had it, I believed it would be appropriate to include it for the rest as well.

Reply:
After a month of non-stop debugging our RHEL9 build I will still disagree :)